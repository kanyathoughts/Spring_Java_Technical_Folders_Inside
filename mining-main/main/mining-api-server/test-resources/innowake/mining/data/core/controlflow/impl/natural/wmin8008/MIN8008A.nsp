DEFINE DATA LOCAL
	1 #value (L)
END-DEFINE
IF #value
   INCLUDE NDT3370CPY1
END-IF

END
