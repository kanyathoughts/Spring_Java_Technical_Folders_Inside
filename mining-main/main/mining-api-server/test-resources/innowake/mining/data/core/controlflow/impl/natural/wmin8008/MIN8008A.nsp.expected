 #2 DefineDataStmt
 #8 IfStmt, Label: IF #value, FROM={TestEntryPoint}, TO={#12(TRUE), #14(FALSE)}
   #12 IncludeStmt, FROM={#8(TRUE)}, TO={#14}
 #14 EndStmt, FROM={#12, #8(FALSE)}, TO={TestReturnPoint}
